Doc/Tutorial/chapter_align.rst

```diff
@@ -576,18 +576,14 @@ alignment are indicated by -1:
     array([ 0,  1,  3,  4,  5, -1, -1]),
     array([0, 1, 2, 3, 4, 5])]
 
+.. _`paragraph:alignment_counts`:
+
 Counting identities, mismatches, and gaps
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``counts`` method calculates the number of identities, mismatches,
-and gaps of a pairwise alignment. For an alignment of more than two
-sequences, the number of identities, mismatches, and gaps are calculated
-and summed for all pairs of sequences in the alignment. The three
-numbers are returned as an ``AlignmentCounts`` object, which is a
-``namedtuple`` with fields ``gaps``, ``identities``, and ``mismatches``.
-This method currently takes no arguments, but in the future will likely
-be modified to accept optional arguments allowing its behavior to be
-customized.
+The ``counts`` method counts the number of identities, mismatches, and gaps
+(insertions and deletions) of an alignment.  The return value is an
+``AlignmentCounts`` object, from which the counts can be obtained as properties.
```

Comment:
So no longer a named tuple, but the old ``.gaps``, ``.identities`` and ``.mismatches`` properties are preserved so writing code which will work before/after this change is simple.